cluster-autoscaler/core/scale_down.go

```diff
@@ -191,6 +211,15 @@ func (sd *ScaleDown) UpdateUnneededNodes(
 		}
 	}
 
+	// Update unremovable map
```

Comment:
Change comment to: add new nodes to unremovable map

Reply:
done